### PR TITLE
MINOR: Avoid NPE when columns cannot be described

### PR DESCRIPTION
**Pull request by gharris1727**
This prevents shadowing table create permission exceptions

Signed-off-by: Greg Harris <gregh@confluent.io>

### CODE REVIEW

**Comment by gharris1727** [comment]
This duplicates #649 and only one of these two should be merged.

File: src/main/java/io/confluent/connect/jdbc/util/TableDefinitions.java [hunk 1]
```diff
@@ -83,8 +83,10 @@ public TableDefinition refresh(
       TableId tableId
   ) throws SQLException {
     TableDefinition dbTable = dialect.describeTable(connection, tableId);
-    log.info("Refreshing metadata for table {} to {}", tableId, dbTable);
-    cache.put(dbTable.id(), dbTable);
+    if (dbTable != null) {
+      log.info("Refreshing metadata for table {} to {}", tableId, dbTable);
+      cache.put(dbTable.id(), dbTable);
+    }
```
Comment (wicknicks) [inline src/main/java/io/confluent/connect/jdbc/util/TableDefinitions.java]: should we log a warn in an else block here saying that the task failed to describe the table? It's hard to follow "absence of logs" when these connectors are typically running against a large number of tables.

Comment (gharris1727) [reply src/main/java/io/confluent/connect/jdbc/util/TableDefinitions.java]: It looks like some of the call-sites log the warning based on the return value, but some don't.
I'll add the log message.
```suggestion
    } else {
      log.warn("Failed to refresh metadata for table {}", tableId);
    }
```
